src/components/ArticleList.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from "react";
+import { fetchArticles } from "../utils/api";
+import Loading from "../components/Loading";
+import Header from "../components/Header";
+import ArticleCard from "./ArticleCard";
+
+const ArticleList = () => {
+  const [articleList, setArticleList] = useState([]);
+  const [isArticlesLoading, setIsArticlesLoading] = useState(true);
+
+  useEffect(() => {
+    fetchArticles()
+      .then((data) => {
+        setIsArticlesLoading(false);
+        setArticleList(data);
+      })
+      .catch((err) => console.log(err));
+  }, []);
+
+  if (isArticlesLoading) {
+    return <Loading isArticlesLoading={isArticlesLoading} />;
+  }
+
+  return (
+    <>
+      <Header />
```

Comment:
This should be above your router in your app, so it get's rendered on every page 😄